docs/source/running.rst

```diff
@@ -101,3 +101,28 @@ While still inside the virtual environment, run::
                  --deployment-dir <contract-deployment-dir> \
                  --source-chain source \
                  --target-chain target
+
+Updating an agent
+-----------------
+
+To describe how to update an agent, it is worth to have a look on Beamer's versioning scheme. As described in
+branching strategy, each major version describes a different mainnet deployment thus a different set of contracts.
```

Comment:
Nitpick: maybe turn `branching strategy` at the beginning into a link to the corresponding doc page?